### ISSUE TITLE
Cloud Practitioner Notes

### ISSUE THREAD

**Issue by obrignoni** [obrignoni/obrignoni.github.io#2]
Cloud Practitioner Notes
------------------------

The Principal element specifies the user, account, service, or other entity that is allowed or denied access to a resource. The bucket policy below has a Principal element set to * which is a wildcard meaning any user. To grant access to a specific IAM user the following format can be used: 

    "Principal": {"AWS":"arn:aws:iam::AWSACCOUNTNUMBER:user/username"}

Direct Connect is used for creating a low-latency private connection to an on-premises data center but it cannot be used to extend the VPC.

AWS Well-Architected helps cloud architects build secure, high-performing, resilient, and efficient infrastructure for their applications and workloads. There are 5 pillars and under the operational excellence pillar the following best practices are recommended: • Perform operations as code • Make frequent, small, reversible changes • Refine operations procedures frequently • Anticipate failure • Learn from all operational failures.

The Amazon Simple Storage Service (S3) offers virtually unlimited storage. The total volume of data and number of objects you can store are unlimited. Individual Amazon S3 objects can range in size from a minimum of 0 bytes to a maximum of 5 terabytes. The largest object that can be uploaded in a single PUT is 5 gigabytes.

An Amazon EC2 Dedicated Host is a physical server fully dedicated for your use, so you can help address corporate compliance requirements. Amazon EC2 Dedicated Hosts allow you to use your eligible software licenses from vendors such as Microsoft and Oracle on Amazon EC2, so that you get the flexibility and cost effectiveness of using your own licenses, but with the resiliency, simplicity and elasticity of AWS.

Amazon Macie is a fully managed data security and data privacy service that uses machine learning and pattern matching to discover and protect your sensitive data in AWS. Amazon Macie automates the discovery of sensitive data at scale and lowers the cost of protecting your data. Macie automatically provides an inventory of Amazon S3 buckets including a list of unencrypted buckets, publicly accessible buckets, and buckets shared with AWS accounts outside those you have defined in AWS Organizations. Then, Macie applies machine learning and pattern matching techniques to the buckets you select to identify and alert you to sensitive data, such as personally identifiable information (PII).
           
Study AWS Macie.
 
Amazon GuardDuty is a threat detection service that continuously monitors for malicious activity and unauthorized behavior to protect your AWS accounts, workloads, and data stored in Amazon S3.
 
Amazon Detective automatically processes terabytes of event data records about IP traffic, AWS management operations, and malicious or unauthorized activity.

AWS WAF can be used to protect on-premises resources if they are deployed behind an Application Load Balancer (ALB). In this scenario the on-premises website servers are added to a target group by IP address. The ALB has a WAF WebACL attached to it and distributes connections to the on-premises website.

Which of the following will help a user determine if they need to request an Amazon EC2 service limit increase? AWS Personal Health Dashboard AWS Cost Explorer AWS Trusted Advisor AWS Service Health Dashboard.
Answer: 3

AWS Trusted Advisor is an online tool that provides you real time guidance to help you provision your resources following AWS best practices. Trusted Advisor checks help optimize your AWS infrastructure, improve security and performance, reduce your overall costs, and monitor service limits.

You can use the consolidated billing feature in AWS Organizations to consolidate billing and payment for multiple AWS accounts. With consolidated billing you get: - One bill for multiple accounts. - Easy tracking or charges across accounts. - Combined usage across accounts and sharing of volume pricing discounts, reserved instance discounts and savings plans. - No extra fee.

Amazon Aurora is a relational database that is compatible with MySQL and PostgreSQL database engines. Aurora is extremely fast and scales up to 128 TB. You can also deploy replicas for read scaling within and across Regions. Aurora also offers automated backups.

Which of the following is a sole responsibility of AWS? Application deployment Patch management Availability Zone management Customer data access controls.
Answer: 3

An Amazon Relational Database Service (RDS) deployment across multiple availability zones is a good example of using the reliability pillar of the AWS Well-Architected Framework. The specific design principle being followed here is “Automatically recover from failure”.

AWS Trusted Advisor offers a rich set of best practice checks and recommendations across five categories: cost optimization, security, fault tolerance, performance, and service limits. The Trusted Advisor “low utilization Amazon EC2 instances” check, checks the Amazon Elastic Compute Cloud (Amazon EC2) instances that were running at any time during the last 14 days and alerts you if the daily CPU utilization was 10% or less and network I/O was 5 MB or less on 4 or more days.

Amazon CloudFront is a content delivery network (CDN) that caches content around the world for lower latency access. AWS Global Accelerator enables access to your application by leveraging the same Edge Locations as CloudFront and routing connections across the AWS global network.

AWS Trusted Advisor can improve the performance of your service by checking your service limits, ensuring you take advantage of provisioned throughput, and monitoring for over-utilized instances.

company is migrating a monolithic application that does not scale well into the cloud and refactoring it into a microservices architecture. Which best practice of the AWS Well-Architected Framework does this plan relate to? Stop spending money on undifferentiated heavy lifting. Implement loosely coupled services. Manage change in automation. Use multiple solutions to improve performance.
Answer: 2

A microservices architecture will help ensure that each component of the application can scale independently and be updated independently. Loose coupling further assists as it places reduces the dependencies between systems and ensures that messages and data being passed between application components can be reliably and durably stored.

You can run Amazon EC2 compute instances hosted on a Snowball Edge with the sbe1, sbe-c, and sbe-g instance types. The sbe1 instance type works on devices with the Snowball Edge Storage Optimized option. The sbe-c instance type works on devices with the Snowball Edge Compute Optimized option. Both the sbe-c and sbe-g instance types work on devices with the Snowball Edge Compute Optimized with GPU option.

If you suspect that AWS resources are used for abusive purposes, contact the AWS Trust & Safety team using the Report Amazon AWS abuse form, or by contacting abuse@amazonaws.com. Provide all the necessary information, including logs in plaintext, email headers, and so on, when you submit your request.

Which on-premises costs must be included in a Total Cost of Ownership (TCO) calculation when comparing against the AWS Cloud? (Select TWO.) Physical compute hardware Operating system administration Network infrastructure in the data center Project management services Database schema development.
Answer: 1, 3

When performing a TCO analysis you must include all costs you are currently incurring in the on- premises environment that you will not pay for in the AWS Cloud. This should include labor costs for activities that will be reduced or eliminated. Labor costs that will continue to be incurred in the cloud need not be included.

A company plan to move the application development to AWS. Which benefits can they achieve when developing and running applications in the AWS Cloud compared to on-premises? (Select TWO.) AWS automatically replicates all data globally. AWS will fully manage the entire application. AWS makes it easy to implement high availability. AWS can accommodate large changes in application demand. AWS takes care of application security patching.
Answer: 3, 4

A cloud practitioner needs to migrate a 70 TB of data from an on-premises data center into the AWS Cloud. The company has a slow and unreliable internet connection. Which AWS service can the cloud practitioner leverage to transfer the data? Amazon S3 Glacier AWS Snowball AWS Storage Gateway AWS DataSync.
Answer: 2

AWS Snowball is a method of transferring the data using a physical device. A Snowball Edge device can hold up to 80 TB so a single device can be used. This transfer method completely avoids the slow and unreliable internet connection.

This is the list of valid IAM best practices: Lock away your AWS account root user access keys Create individual IAM users Use groups to assign permissions to IAM users Grant least privilege Get started using permissions with AWS managed policies Use customer managed policies instead of inline policies Use access levels to review IAM permissions Configure a strong password policy for your users Enable MFA Use roles for applications that run on Amazon EC2 instances Use roles to delegate permissions Do not share access keys Rotate credentials regularly Remove unnecessary credentials Use policy conditions for extra security Monitor activity in your AWS account Video presentation about IAM best practices

A Service Control Policy (SCP) is used to manage the maximum available permissions and is associated with which of the following? Service control policies (SCPs) manage permissions for which of the following? AWS Global Infrastructure AWS Regions AWS Organizations Availability Zones.
Answer: 3

Service control policies (SCPs) are a type of organization policy that you can use to manage permissions in your organization. SCPs offer central control over the maximum available permissions for all accounts in your organization. SCPs are associated with AWS Organizations and help you to ensure your accounts stay within your organization’s access control guidelines. SCPs are available only in an organization that has all features enabled.

What should a Cloud Practitioner ensure when designing a highly available architecture on AWS? Servers have low-latency and high throughput network connectivity. The failure of a single component should not affect the application. There are enough servers to run at peak load available at all times. A single monolithic application component handles all operations. Answer: 2

In a highly available system the failure of a single component should not affect the application. This means that if a single component such as an application server fails, there should be other applications servers available that can seamlessly take over operations and ensure the application continues to operate.

AWS Virtual Private Network solutions establish secure connections between your on-premises networks, remote offices, client devices, and the AWS global network.
 
Amazon Elastic Map Reduce (EMR) is a web service that enables businesses, researchers, data analysts, and developers to easily and cost-effectively process vast amounts of data. EMR utilizes a hosted Hadoop framework running on Amazon EC2 and Amazon S3.

AWS Personal Health Dashboard provides alerts and remediation guidance when AWS is experiencing events that may impact you. While the Service Health Dashboard displays the general status of AWS services, Personal Health Dashboard gives you a personalized view into the performance and availability of the AWS services underlying your AWS resources. The dashboard displays relevant and timely information to help you manage events in progress, and provides proactive notification to help you plan for scheduled activities. With Personal Health Dashboard, alerts are triggered by changes in the health of AWS resources, giving you event visibility, and guidance to help quickly diagnose and resolve issues.

Which AWS service should a Cloud Practitioner use to automate configuration management using Puppet? AWS Config AWS OpsWorks AWS CloudFormation AWS Systems Manager
Answer: 2
OpsWorks?

AWS OpsWorks is a configuration management service that provides managed instances of Chef and Puppet. Chef and Puppet are automation platforms that allow you to use code to automate the configurations of your servers. OpsWorks lets you use Chef and Puppet to automate how servers are configured, deployed, and managed across your Amazon EC2 instances or on-premises compute environments.

Amazon SNS provides topics for high-throughput, push-based, many-to-many messaging. Using Amazon SNS topics, your publisher systems can fan out messages to a large number of subscriber endpoints for parallel processing, including Amazon SQS queues, AWS Lambda functions, and HTTP/S webhooks. Additionally, SNS can be used to fan out notifications to end users using mobile push, SMS, and email.

Amazon Inspector is an automated security assessment service that helps improve the security and compliance of applications deployed on AWS. Amazon Inspector automatically assesses applications for exposure, vulnerabilities, and deviations from best practices. After performing an assessment, Amazon Inspector produces a detailed list of security findings prioritized by level of severity. These findings can be reviewed directly or as part of detailed assessment reports which are available via the Amazon Inspector console or API.

An AWS Direct Connect connection is a private, dedicated link to AWS. As it does not use the internet, performance is consistent. The following diagram shows how a corporate data center is connected to AWS using a Direct Connect link via an AWS Direct Connect location:

The AWS CloudHSM service helps you meet corporate, contractual, and regulatory compliance requirements for data security by using dedicated Hardware Security Module (HSM) instances within the AWS cloud. AWS CloudHSM enables you to easily generate and use your own encryption keys on the AWS Cloud.

Highlight (Yellow) and Note | Location 3481
What are the benefits of using the AWS Managed Services? (Select TWO.) Alignment with ITIL processes Managed applications so you can focus on infrastructure Baseline integration with ITSM tools Designed for small businesses Support for all AWS services
Answer: 1,3
ITIL? ITSM?

AWS Managed Services manages the daily operations of your AWS infrastructure in alignment with ITIL processes. AWS Managed Services provides a baseline integration with IT Service Management (ITSM) tools such as the ServiceNow platform. AWS Managed Services provides ongoing management of your AWS infrastructure so you can focus on your applications. By implementing best practices to maintain your infrastructure, AWS Managed Services helps to reduce your operational overhead and risk. AWS Managed Services currently supports the 20+ services most critical for Enterprises, and will continue to expand our list of integrated AWS services. AWS Managed Services is designed to meet the needs of Enterprises that require stringent SLAs, adherence to corporate compliance, and integration with their systems and ITIL®-based processes.

What are the names of two types of AWS Storage Gateway? (Select TWO.) S3 Gateway File Gateway Block Gateway Tape Gateway Cached Gateway
Answer: 2,4

The AWS Storage Gateway service enables hybrid storage between on-premises environments and the AWS Cloud. It provides low-latency performance by caching frequently accessed data on premises, while storing data securely and durably in Amazon cloud storage services. AWS Storage Gateway supports three storage interfaces: file, volume, and tape File gateway provides a virtual on-premises file server, which enables you to store and retrieve files as objects in Amazon S3 The volume gateway represents the family of gateways that support block-based volumes, previously referred to as gateway-cached and gateway-stored modes Tape Gateway (formerly known as Gateway Virtual Tape Library) is used for backup with popular backup software.

Amazon S3 Standard-Infrequent Access is the most cost-effective choice. It provides immediate access and is suitable for this use case as it is lower cost than S3 standard. Note that you must pay a fee for retrievals which is why you would only use this tier for infrequent access use cases.

Which AWS service or feature can be used to restrict the individual API actions that users and roles in each member account can access? Amazon Macie AWS Organizations AWS Shield AWS IAM Answer: 2

AWS Organizations offers Service control policies (SCPs) which are a type of organization policy that you can use to manage permissions in your organization. SCPs offer central control over the maximum available permissions (API actions) for all accounts in your organization. SCPs help you to ensure your accounts stay within your organization’s access control guidelines. SCPs are available only in an organization that has all features enabled.

Amazon CloudWatch is a performance monitoring tool that receives metrics from AWS services. This data can be used for monitoring the operational health of resources as well as being used to optimize costs through ensuring systems are right-sized and just enough capacity is provisioned.

Only the enterprise plan provides Well-Architected Reviews and Operational Reviews. 24/7 access to Cloud Support Engineers through email, online chat, and phone is offered on the business and enterprise plans.

Under the AWS shared responsibility model what is AWS responsible for? (Select TWO.) Physical security of the data center Replacement and disposal of disk drives Configuration of security groups Patch management of operating systems Encryption of customer data Answer: 1,2

When using AWS Organizations with consolidated billing, best practices include: – Always enable multi-factor authentication (MFA) on the root account. – Always use a strong and complex password on the root account. – The Paying account should be used for billing purposes only. Do not deploy resources into the Paying account. There is a default limit of 20 linked accounts but this can be extended and there is no reason why you should stick to a maximum of 20 accounts.

You need to run a production process that will use several EC2 instances and run constantly on an ongoing basis. The process cannot be interrupted or restarted without issue. What EC2 pricing model would be best for this workload? Reserved instances Spot instances On-demand instances Flexible instances Answer: 1

Which database allows you to scale at the push of a button without incurring any downtime? Amazon RDS Amazon EMR Amazon DynamoDB Amazon RedShift.
Answer: 3

AWS Direct Connect is a network service that provides an alternative to using the Internet to connect customers’ on premise sites to AWS. Data is transmitted through a private network connection between AWS and a customer’s data center or corporate network. Benefits of AWS Direct Connect: – Reduce cost when using large volumes of traffic. – Increase reliability (predictable performance). – Increase bandwidth (predictable bandwidth). – Decrease latency.

Amazon Neptune is a fast, reliable, fully-managed graph database service that makes it easy to build and run applications that work with highly connected datasets. With Amazon Neptune, you can create sophisticated, interactive graph applications that can query billions of relationships in milliseconds.

When using Amazon IAM, what authentication methods are available to use? (Select TWO.) Client certificates Access keys AWS KMS Server certificates AES 256.
Answer: 2,4

Supported authentication methods include console passwords, access keys and server certificates. Access keys are a combination of an access key ID and a secret access key and can be used to make programmatic calls to AWS. Server certificates are SSL/TLS certificates that you can use to authenticate with some AWS services.     
 
Which of the below AWS services supports automated backups as a default configuration? Amazon S3 Amazon RDS Amazon EC2 Amazon EBS Answer: 2

What are the advantages of Availability Zones? (Select TWO.) They allow regional disaster recovery They provide fault isolation They enable the caching of data for faster delivery to end users They are connected by low-latency network connections They enable you to connect your on-premises networks to AWS to form a hybrid cloud.
Answer: 2,4

Each AWS region contains multiple distinct locations called Availability Zones (AZs). Each AZ is engineered to be isolated from failures in other AZs. An AZ is a data center, and in some cases, an AZ consists of multiple data centers. AZs within a region provide inexpensive, low-latency network connectivity to other zones in the same region. This allows you to replicate your data across data centers in a synchronous manner so that failover can be automated and be transparent for your users.

Which of the options below are recommendations in the cost optimization pillar of the well-architected framework? (Select TWO.) Adopt a consumption model Adopt a capital expenditure model Start spending money on data center operations Analyze and attribute expenditure Manage your services independently.
Answer: 1,4

To ensure the security of your AWS account, what are two AWS best practices for managing access keys? (Select TWO.) Don’t create any access keys, use IAM roles instead Don’t generate an access key for the root account user Where possible, use IAM roles with temporary security credentials Rotate access keys daily Use MFA for access keys.
Answer: 2,3

Best practices include: – Don’t generate an access key for the root account user. – Use Temporary Security Credentials (IAM Roles) Instead of Long-Term Access Keys. – Manage IAM User Access Keys Properly.

An Amazon EC2 instance running the Amazon Linux 2 AMI is billed in what increment? Per second Per hour Per CPU Per GB.
Answer: 1

Reservations provide you with greater discounts, up to 75%, by paying for capacity ahead of time. Some of the services you can reserve include: EC2, DynamoDB, ElastiCache, RDS, and RedShift.

Which service allows an organization to view operational data from multiple AWS services through a unified user interface and automate operational tasks? AWS Config AWS OpsWorks AWS Systems Manager Amazon CloudWatch.
Answer: 3

AWS Systems Manager gives you visibility and control of your infrastructure on AWS. Systems Manager provides a unified user interface so you can view operational data from multiple AWS services and allows you to automate operational tasks across your AWS resources.

AWS Config is a fully-managed service that provides you with an AWS resource inventory, configuration history, and configuration change notifications to enable security and regulatory compliance. CORRECT: "Configure AWS Config with the resource types" is the correct answer.

Amazon GuardDuty offers threat detection and continuous security monitoring for malicious or unauthorized behavior to help you protect your AWS accounts and workloads.

What is a specific benefit of an Enterprise Support plan? Included Technical Support Manager Included AWS Solutions Architect Included Cloud Support Associate Included Technical Account Manager.
Answer: 4

Only the Enterprise Support plan gets a Technical Account Manager (TAM). You do not get an AWS Solutions Architect with any plan. Cloud Support Associates are provided in the Developer plan. There’s no such thing as a Technical Support Manager in the AWS support plans.

Which of the following are advantages of the AWS Cloud? (Select TWO.) AWS manages the maintenance of the cloud infrastructure AWS manages the security of applications built on AWS AWS manages capacity planning for physical servers AWS manages the development of applications on AWS AWS manages cost planning for virtual servers.
Answer: 1, 3

A user has an AWS account with a Business-level AWS Support plan and needs assistance with handling a production service disruption. Which action should the user take? Contact the dedicated Technical Account Manager Contact the dedicated AWS Concierge Support team Open a business-critical system down support case Open a production system down support case.
Answer: 4

Which type of AWS Storage Gateway can be used to backup data with popular backup software? File Gateway Volume Gateway Gateway Virtual Tape Library Backup Gateway.
Answer: 3

What billing timeframes are available for Amazon EC2 on-demand instances? (Select TWO.) Per week Per day Per hour Per minute Per second.
Answer: 3,5

When performing a total cost of ownership (TCO) analysis between on-premises and the AWS Cloud, which factors are only relevant to on-premises deployments? (Select TWO.) Hardware procurement teams Operating system licensing Facility operations costs Database administration Application licensing.
Answer: 1,3

AWS CloudHSM is a cloud-based hardware security module (HSM) that allows you to easily add secure key storage and high-performance crypto operations to your AWS applications.

To reduce the price of your Amazon EC2 instances, which term lengths are available for reserved instances? (Select TWO.) 4 years 1 year 5 years 2 years 3 years.
Answer: 2,5

Reserved instances provide significant discounts, up to 75% compared to On-Demand pricing, by paying for capacity ahead of time. They are good for applications that have predictable usage, that need reserved capacity, and for customers who can commit to a 1 or 3-year term.

Where can resources be launched when configuring Amazon EC2 Auto Scaling? A single subnet Multiple AZs within a region Multiple AZs and multiple regions Multiple VPCs.
Answer: 2

You can configure subnets and endpoints within the VPC section of AWS management console. EBS volumes and ELB must be configured in the EC2 section of the AWS management console and DNS records must be configured in Amazon Route 53.

In addition to DNS services, what other services does Amazon Route 53 provide? (Select TWO.) DHCP Caching Domain registration IP Routing Traffic flow.
Answer: 3,5

Amazon Route 53 features include domain registration, DNS, traffic flow, health checking, and failover. .Route 53 does not support DHCP, IP routing or caching.

AWS customers are welcome to carry out security assessments or penetration tests against their AWS infrastructure without prior approval for the following eight services: Amazon EC2 instances, NAT Gateways, and Elastic Load Balancers. Amazon RDS. Amazon CloudFront. Amazon Aurora. Amazon API Gateways. AWS Lambda and Lambda Edge functions. Amazon LightSail resources. Amazon Elastic Beanstalk environments.

Which team is available to support AWS customers on an Enterprise support plan with account issues? AWS Technical Support AWS Billing and Accounts AWS Concierge AWS Technical Account Manager.
Answer: 3

The reliability pillar includes the ability of a system to recover from infrastructure or service disruptions, dynamically acquire computing resources to meet demand, and mitigate disruptions such as misconfigurations or transient network issues There are five design principles for reliability in the cloud: – Test recovery procedures. – Automatically recover from failure. – Scale horizontally to increase aggregate system availability. – Stop guessing capacity. – Manage change in automation.

Which feature of AWS IAM enables you to identify unnecessary permissions that have been assigned to users? Role Advisor Access Advisor Permissions Advisor Group Advisor.
Answer: 2
Access Advisor

The IAM console provides information about when IAM users and roles last attempted to access AWS services. This information is called service last accessed data. This data can help you identify unnecessary permissions so that you can refine your IAM policies to better adhere to the principle of “least privilege.” That means granting the minimum permissions required to perform a specific task. You can find the data on the Access Advisor tab in the IAM console by examining the detail view for any IAM user, group, role, or managed policy.

The five pillars of the AWS Well-Architected Framework are operational excellence, security, reliability, performance efficiency, and cost optimization.

Both S3 and DynamoDB automatically scale as demand dictates. In the case of DynamoDB you can either configure the on-demand or provisioned capacity mode. With on-demand capacity mode DynamoDB automatically adjusts the read and write throughput for you. EBS and RDS do not scale automatically. You must intervene to adjust volume sizes and database instance types to scale these resources.
            
Which AWS components aid in the construction of fault-tolerant applications? (Select TWO.) Elastic IP addresses ARNs AMIs Tags Block device mappings.
Answer: 1, 3

Elastic IP addresses can be easily remapped between EC2 instances in the event of a failure. Amazon Machine Images (AMIs) can be used to quickly launch replacement instances when there is a failure Amazon Resource Names (ARNs), tags and block device mappings don’t really help with fault tolerance.

The public cloud is offered under a purely pay as you go model (unless you choose to reserve), and allows companies to completely avoid CAPEX costs. The public cloud is also highly elastic so companies can grow and shrink the applications as demand changes. Private and on-premise clouds are essentially the same, though both could be managed by a third party and even could be delivered under an OPEX model by some vendors. However, they are typically more CAPEX heavy and the elasticity is limited.

Which of the following are advantages of using the AWS cloud computing over legacy IT? (Select TWO.) You are able to pass responsibility for the availability of your application to AWS You don't need to worry about over provisioning as you can elastically scale You don't need to patch your operating systems You can bring new applications to market faster You can bring services closer to your end users.
Answer: 2, 4

With cloud computing you no longer need to guess about capacity as you can elastically scale. This means you don’t end up over-provisioning but instead react to the load on your servers. You can also be faster and more agile with development and release of applications.

AWS Lambda and Amazon API Gateway are both app-facing components of the AWS Serverless infrastructure AWS Step Functions is an orchestration service.

Amazon Elastic Container Registry (ECR) is a fully-managed Docker container registry that makes it easy for developers to store, manage, and deploy Docker container images. Amazon ECR is integrated
with Amazon Elastic Container Service (ECS). Amazon ECR eliminates the need to operate your own container repositories or worry about scaling the underlying infrastructure.

With AWS Lambda you don’t have any servers to manage (serverless). Lambda functions scale out rather than up running multiple invocations of the function in parallel.

What offerings are included in the Amazon Lightsail product set? (Select TWO.) Virtual Private Server NoSQL database Managed MySQL database Object storage Serverless functions.
Answer: 1, 3

Amazon LightSail provides an easy, low cost way to consume cloud services without needing the skill set for using VPC resources. The product set includes virtual private servers (instances), managed MySQL databases, HA storage, and load balancing You can connect to other AWS services such as S3, DynamoDB, and CloudFront, however these are not part of the LightSail product range.

Amazon SageMaker is a fully-managed platform that enables developers and data scientists to quickly and easily build, train, and deploy machine learning models at any scale. Amazon SageMaker removes all the barriers that typically slow down developers who want to use machine learning.

Amazon Route 53 health checks monitor the health and performance of your web applications, web servers, and other resources. None of the other options provide a solution that can check the health and performance of an application.
   
How can a company connect from their on-premises network to VPCs in multiple regions using private connections? AWS Managed VPN AWS Direct Connect Gateway Amazon CloudFront Inter-Region VPC Peering Answer: 2 You can use an AWS Direct Connect gateway to connect your AWS Direct Connect connection over a private virtual interface to one or more VPCs in your account that are located in the same or different Regions.

When designing a VPC, what is the purpose of an Internet Gateway? Provides Internet access for EC2 instances in private subnets Enables Internet communications for instances in public subnets It's a bastion host for inbound management connections It's used for making VPN connections to a VPC Answer: 2 An internet gateway is a horizontally scaled, redundant, and highly available VPC component that allows communication between instances in your VPC and the internet. It therefore imposes no availability risks or bandwidth constraints on your network traffic. An internet gateway serves two purposes: to provide a target in your VPC route tables for internet-routable traffic, and to perform network address translation (NAT) for instances that have been assigned public IPv4 addresses.

Amazon EBS encryption offers a straight-forward encryption solution for your EBS resources that doesn't require you to build, maintain, and secure your own key management infrastructure. It uses AWS Key Management Service (AWS KMS) customer master keys (CMK) when creating encrypted volumes and snapshots. Encryption operations occur on the servers that host EC2 instances, ensuring the security of both data- at-rest and data-in-transit between an instance and its attached EBS storage. All volumes can now be encrypted at launch time and it’s possible to set this as the default setting.
